lib/src/postgrest.dart

```diff
@@ -23,7 +25,8 @@ class PostgrestClient {
     Map<String, String>? headers,
     this.schema,
     this.httpClient,
-  }) : headers = {...defaultHeaders, if (headers != null) ...headers};
+  })  : headers = {...defaultHeaders, if (headers != null) ...headers},
+        _isolate = PostgrestIsolate()..init();
```

Comment:
What if we make it so that we can pass `PostgrestIsolate` as parameters like this:

```dart
PostgrestClient(
    this.url, {
    Map<String, String>? headers,
    this.schema,
    this.httpClient,
    PostgrestIsolate? isolate,
    }): headers = {...defaultHeaders, if (headers != null) ...headers},
        _isolate = (isolate ?? PostgrestIsolate()); // Call `init()` somewhere else
```

Then we could add `PostgrestIsolate` property on `SupabaseClient` to persist an isolate object and just pass that value whenever we call `from` or `rpc`.

Reply:
Sounds like a great idea